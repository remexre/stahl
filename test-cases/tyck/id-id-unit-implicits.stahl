module test
  test

defty Unit TYPE
  unit Unit

def id
  pi* ((T TYPE)) (pi ((x T)) T)
  fn* (_) (fn (x) x)

def test
  id id unit
