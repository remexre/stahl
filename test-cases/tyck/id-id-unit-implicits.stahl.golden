#std:prelude:unit#
#std:prelude:Unit#
